eva-accession-import/pom.xml

```diff
@@ -22,6 +22,10 @@
             <artifactId>eva-accession-core</artifactId>
             <version>0.1</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.fakemongo</groupId>
+            <artifactId>fongo</artifactId>
+        </dependency>
```

Comment:
Given that Fongo already causes issues during testing and we are using a real Mongo database in other tests, do we really want to keep it for some particular classes?

Reply:
given that it's not that important for this module, let's not hurry and see if the fixed fongo gets deployed https://github.com/fakemongo/fongo/issues/343#issuecomment-402500418

Reply:
I didn't know someone had recently addressed the issue, I'm fine with waiting then.